content/en/blog/2021-01-21-hello-world.md

```diff
@@ -5,7 +5,7 @@ date: "2021-01-21T16:20:18-04:00"
 draft: false
 image: img/blog/2021-01-21-hello-world/gulfatnight.jpg
 author: Benjamin Bengfort
-category: "Distributed Systems"
+tags: ['Distributed Systems']
```

Comment:
Hugo has a built in support for taxonomies to group content. Tags and categories are defined by default. In order for it to work, we have to make the taxonomy plural in the frontmatter. We can create a custom taxonomies, so we could make this `blog-tags` if we want.

I thought `/tags/tag-name` would be shorter, so opted for that over `categories/category-name`. Here's a link to more info about taxonomies: https://gohugo.io/content-management/taxonomies/#default-taxonomies